descwl/analysis.py

```diff
@@ -901,7 +908,7 @@ def finalize(self,verbose,trace,calculate_bias):
             data['ri_color'][index] = model.ri_color
             data['flux'][index] = model.model.flux
             # Is this galaxy's centroid visible in the survey image?
-            data['visible'][index] = 1 if self.survey.image.bounds.includes(bounds.center.pos) else 0
+            data['visible'][index] = 1 if self.survey.image.bounds.includes(bounds.center.x, bounds.center.y) else 0
```

Comment:
yup

Reply:
I've also modified `survey.py` so that no_analysis let's you skip PSF moment computation of the PSF from high res image and `analysis.py` to return catalog without fisher entries. I'm not sure if this can be done just with a function in `render.py`